password.php

```diff
@@ -41,6 +41,12 @@
     exit;
 }
 
+if (isset($_GET['returnpasswords']) && $_GET['returnpasswords'] == 1) {
```

Comment:
raw access to $_GET is not really allowed - can you please replace this with optional_param